test/swift_class_test.exs

```diff
@@ -271,17 +271,32 @@ defmodule SwiftClassTest do
       """
 
       output = [
-        {{:<>, [context: Elixir, imports: [{2, Kernel}]], ["color-", {:color_name, [], Elixir}]},
+        {[{:<>, [context: Elixir, imports: [{2, Kernel}]], ["color-", {:color_name, [], Elixir}]}, {:_target, [], Elixir}],
          [
            {:foo, [], [true]},
-           {:color, [], [{Elixir, [], {:color_name, [], Elixir}}]},
+           {:color, [], [{:color_name, [], Elixir}]},
            {:bar, [], [false]}
          ]},
         {
-          "color-red",
-          [
-            {:color, [], [{:., [], [nil, :red]}]}
-          ]
+          ["color-red", {:_target, [], Elixir}],
+          [{:color, [], [{:., [], [nil, :red]}]}]
+        }
+      ]
+
+      assert parse_class_block(input) == output
+    end
+
+    test "can take optional target in definition" do
+      input = """
+        "color-red", target: :watchos do
+          color(.red)
+        end
+      """
+
+      output = [
+        {
+          ["color-red", [target: :watchos]],
+          [{:color, [], [{:., [], [nil, :red]}]}]
```

Comment:
The block is missing here

Reply:
yes you caught me mid change, I've removed that syntax. See my comments.

Reply:
tldr; `__block__` are multiple expressions so in the resulting function would only return the last expression. We need all returned so they need to be in a `List`